.drone.yml

```diff
@@ -144,10 +144,24 @@ steps:
       IGNORE_UNFIXED: true
       ALLOW_CVE_LIST_FILE: hof-services-config/Change_Of_Address/trivy-cve-exceptions.txt
     when:
-      event:
-      - pull_request
-      - push
-      - tag
+      event: [push, pull_request, tag]
```

Comment:
why would we want to run these checks on tag event?

Reply:
@dk4g  We are using this condition for the events as suggested in ACP documentation. In future if we use tags for versioning our product/services. It can run the pipeline. 
We can remove it if you or team thinks this doesnt help us in anyway

Reply:
As we have the conversation about Trivy yesterday. The Trivy steps are missing the failing of the pipeline when agreed in the criticality. Also, we need a master folder to add all projects trivy exceptions file.